content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-core.md

```diff
@@ -61,7 +61,7 @@ For a full list of Datadog's .NET Core library and processor architecture suppor
 </div>
 
 <div class="alert alert-warning">
-  <strong>Note:</strong> Datadog's automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, Datadog's .NET Tracer with Profiler enabled). To ensure maximum visibility, run only one APM solution in your application environment.
+  <strong>Note:</strong> Datadog's automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber, with exception of our Datadog APM and the Continous Profiler. To ensure maximum visibility, run only one APM solution in your application environment. 
```

Comment:
```suggestion
  <strong>Note:</strong> Datadog's automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber, with the exception of APM and the Continuous Profiler. To ensure maximum visibility, run only one APM solution in your application environment. 
```

Reply:
What do you think of this suggestion? I think it might be confusing to say that there is are exceptions. Additionally, whatever we end up with in this file, can you please copy the contents over to the dotnet-framework page as well?
```suggestion
  <strong>Note:</strong> Datadog's automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, Datadog APM). To ensure maximum visibility, run only one APM solution in your application environment. 
```

Reply:
We can go with that to keep it simple. I agree.